app/views/onboard/_includes/guidance.njk

```diff
@@ -1,19 +1,19 @@
 
 {% set guidanceHtml %}
-  <h2 class="govuk-heading-m">Making decisions</h2>
-  <p>You can let the organisation make offers, amend offers and reject applications.</p>
-  <h2 class="govuk-heading-m">Access to safeguarding information</h2>
-  <p>You can give the organisation access to:</p>
+  <h2 class="govuk-heading-m">Permission to make decisions</h2>
+  <p>You can choose to allow this organisation to make offers, amend offers and reject applications.</p>
```

Comment:
Is the ‘choose’ not implied by the preceding ‘can’?

Reply:
Yes! I think it is. I'll make that change